migrations/000048_HealthAuthorityStats.up.sql

```diff
@@ -0,0 +1,71 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE HealthAuthorityStatsRelease(
+    health_authority_id INT NOT NULL REFERENCES HealthAuthority(id),
+    not_before TIMESTAMPTZ NOT NULL,
+    -- timeone name, i.e. America/New_York
+    timezone TEXT,
```

Comment:
I'm not sure I'm following the timezone fields here.

Reply:
the idea is that this server will always track stats in UTC, segmented by hour.

The client can request a projection onto a specific timezone (not receiving the hourly breakdowns).

And then we want to prevent any health authority from rapidly rotating through timezone projections to derive individual hour segments

Reply:
removed - we will do UTC everywhere